### ISSUE TITLE
Copy Note as Rich Text

### ISSUE THREAD

**Issue by vHanda** [GitJournal/GitJournal#215]
Currently when sharing a note, it is sent as plain text, and the markdown widget does not let the text be selected. It would be nice if there was some way of copying the note with the formatting.

**Comment by relston** [comment]
Yes, I think the use case for me is being able to share a note in an email with all the formatting or into a Google Doc.

**Comment by vHanda** [comment]
Blocked by https://github.com/flutter/flutter/issues/23603
